sns_ik_lib/include/sns_ik/sns_position_ik.hpp

```diff
@@ -110,6 +111,24 @@ class SNSPositionIK {
                               double* errR,
                               KDL::Vector* trans,
                               KDL::Vector* rotAxis);
+
+    /**
+     * determines the rotation axis necessary to rotate from frame b1 to the
+     * orientation of frame b2 and the vector necessary to translate the origin
+     * of b1 to the origin of b2, and stores the result in a Twist
+     * datastructure.  The result is w.r.t. frame b1.
+     * \param F_a_b1 frame b1 expressed with respect to some frame a.
+     * \param F_a_b2 frame b2 expressed with respect to some frame a.
+     * \warning The result is not a real Twist!
+     * \warning In contrast to standard KDL diff methods, the result of
+     * diffRelative is w.r.t. frame b1 instead of frame a.
+     */
```

Comment:
This follows the change made in trac-ik:
https://bitbucket.org/traclabs/trac_ik/commits/6994051